documentation/src/main/asciidoc/topics/con_local_caches.adoc

```diff
@@ -8,6 +8,18 @@ write-behind to persistent storage as well as management capabilities such as ev
 
 The {brandname} `Cache` API extends the `ConcurrentMap` API in Java, making it easy to migrate from a map to a {brandname} cache.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
+This restriction reduces server failure issues that can arise with long cache names or cache template names.
```

Comment:
The restriction was because an internal limitation (in the byte representation we use during the cache creation IIRC) that will always fail with names longer than 255 bytes. We could rephrase this or remove.

Reply:
Hey @jabolina . Thanks for reviewing the content and for adding feedback. 

My meaning behind the restriction was more focused on why this restriction has come into play. Might I ask what was the previous behavior? Why a character limit of 255 is required. What happened when a user created a cache and named it with + 255 characters?

Reply:
The limitation already existed due to the way we represented the name during the creation. 

https://github.com/infinispan/infinispan/blob/22d7148c5b63d262f7bd0bb33d0c4f97de38a3a6/core/src/main/java/org/infinispan/util/ByteString.java#L36-L38

Originally an unchecked exception was thrown, which could crash the server, with the change we kind of formalized this limit, now we throw an exception with a proper feedback and explanation.